### PR TITLE
OOP school library: add basic UI

### PR DESCRIPTION
**Pull request by vanina123**
- Build interactive console apps.
- Run a program using the command line.
- No linters error
- Flow git flows
- Document work in a professional way
- Use Ruby best practices

### CODE REVIEW

File: main.rb [hunk 1]
```diff
@@ -1,20 +1,63 @@
...
-require_relative 'student'
-require_relative 'teacher'
-require_relative 'nameable'
+require_relative 'app'
+def display
+  {
+    1 => 'list_all_books',
+    2 => 'list_all_people',
+    3 => 'create_a_teacher',
+    4 => 'create_a_student',
```
Comment (tugsem) [inline main.rb]: - [ ] According to the wireframe, there must be one option 'create a person'. Then you need to ask the user if to create a teacher or a student and get user input that way and return the required methods afterward. Kindly apply it.
image
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @people.each do |person|
+      puts "ID: #{person.id}, Name: #{person.name}"
+    end
+  end
```
Comment (tugsem) [inline app.rb]: - [ ] According to the wireframe, you need to return these properties as well, kindly use this format:
 `[Student/Teacher] Name: Name, ID: ID, Age: Age`
given:
image
yours:
image

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+  def create_a_rental
+    puts 'Choose a book from the list by number'
+    @books.each_with_index do |book, index|
+      puts "#{index} - #{book.title} by #{book.author}"
+    end
+    book_index = gets.chomp.to_i
+    book = @books[book_index]
+    puts 'Choose a person from the list by number:'
+    @people.each_with_index do |person, index|
+      puts "#{index} - #{person.name} - #{person.age} - #{person.class}"
```
Comment (tugsem) [inline app.rb]: - [ ] According to the wireframe you need to return the people in this format. Kindly apply it. 
image
your app:
image

File: main.rb [hunk 4]
```diff
@@ -1,20 +1,63 @@
...
+    5 => 'create_a_book',
+    6 => 'create_a_rental',
+    7 => 'list_rentals_for_a_person',
```
Comment (tugsem) [inline main.rb]: - [ ] Please change this option to 'List all rentals for a given person id' according to the wireframe. And you need to ask for an id next, please change this method to be exactly like this: 
image
your app:
image

File: app.rb [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+
+  def list_rentals_for_a_person
+    puts 'Choose a person from the list by number'
+    @people.each_with_index do |person, index|
+      puts "#{index} - #{person.name}"
+    end
+    person_index = gets.chomp.to_i
+    person = @people[person_index]
+    person.rental.each do |rental|
+      puts "Book: #{rental.book.title} - Date: #{rental.date}"
```
Comment (tugsem) [inline app.rb]: - [ ] According to the wireframe, rentals must be seen like this format. Please fix it. 
image
your app: 
image

**Review by tugsem — CHANGES_REQUESTED** [review]
##  STATUS: REQUIRED CHANGES ♻️
Hello  @vanina123 👋 , 

Good job so far! 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!

Alt Text
## Highlights 🚀 
✔️app works correctly
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @vanina123, 

To highlight 👍🏽 
✔️ App runs smoothly
✔️ App matches wireframe
✔️ App UX is good

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**